### ISSUE TITLE
Error on Ubuntu 17.10

### ISSUE THREAD

**Issue by cogitas3d** [wwmm/pulseeffects#137]
Hey there!

I installed pulseeffects in Ubuntu 17.10 and appear this error when I run it;

`** (pulseeffects:3019): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-nVlI1rVw0B: Conexão recusada
Traceback (most recent call last):
  File "/usr/bin/pulseeffects", line 6, in <module>
    a = Application()
  File "/usr/lib/python3/dist-packages/PulseEffects/application.py", line 36, in __init__
    self.gst = GstEffects()
  File "/usr/lib/python3/dist-packages/PulseEffects/gst.py", line 49, in __init__
    self.pipeline = self.build_pipeline()
  File "/usr/lib/python3/dist-packages/PulseEffects/gst.py", line 91, in build_pipeline
    self.audio_src.set_property('client-name', 'PulseEffects')
AttributeError: 'NoneType' object has no attribute 'set_property'`

I read bout it, I installed python3-gst-1.0 but the error continue.

**Comment by wwmm** [comment]
Did you install the packages suggested in the wiki? [https://github.com/wwmm/pulseeffects/wiki/Installation-from-Source](https://github.com/wwmm/pulseeffects/wiki/Installation-from-Source)

This error is happening because you do not have the gstreamer plugin `pulsesrc` installed.

**Comment by wwmm** [comment]
You could also try to use a third party repository [https://github.com/wwmm/pulseeffects/wiki/Package-Repositories](https://github.com/wwmm/pulseeffects/wiki/Package-Repositories)

**Comment by cogitas3d** [comment]
Hi!

All right! I installed:

`sudo apt-get install gstreamer1.0-pulseaudio`

And all run well!

THank you very much!
